gini-swift-demo-ios/Info.plist

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
```

Comment:
I can't see the saved original and rectified images in iTunes. `UIFileSharingEnabled` missing?